hazelcast-sql/src/main/java/com/hazelcast/sql/impl/calcite/validate/operators/predicate/HazelcastInOperator.java

```diff
@@ -149,5 +154,8 @@ private static boolean anyNullable(List<RelDataTypeField> fieldList) {
     interface HazelcastInOperatorResource extends CalciteResource {
         @Resources.BaseMessage("Sub-queries are not allowed for IN operator.")
```

Comment:
The new message says "not supported", it would be better to say that here too:
```suggestion
        @Resources.BaseMessage("Sub-queries are not supported for IN operator.")
```